### DISCUSSION TITLE
Which headers go where? V1.2

### DISCUSSION THREAD

**Discussion by razidan** [alanbjohnston/CubeSatSim#312] [Q&A] [answered]
Hello!

After a previous unsuccessful attempt, I'm trying to understand which stacking headers go on which board.
I know the non stacking headers are on the STEM board. I'm not sure which of these goes on Battery/Main board (to get the spacing perfect).
On the BOM, both the X-long and the X-tall have the same reference (J1).

Thanks! 

![headers](https://github.com/alanbjohnston/CubeSatSim/assets/78091655/d5e61ad8-373e-432c-9b9b-695b34c1a69a)

**Comment by alanbjohnston** [comment, marked as answer]
Hi Razan,

This answer is for the white v1.2 PCBs. The new beta v1.3 blue boards use all the same stacking headers, which simplifies things. 

For the v1.2 white PCBs, the stacking headers soldered onto the PCBs are the extra-long ones (left in your photo). The stacking headers used as extenders between the boards (not soldered to anything) are the ones on the right in your photo. 

However, if you get it wrong, nothing breaks, you just have to play around with the standoffs between the boards so the boards are fully plugged in and supported. 

If you use the right ones, the standoff combinations in the instructions should work as is. 

Good luck!
Alan

**Reply by razidan** [reply]
Thank you! 
Will try this right now!

**Reply by razidan** [reply]
BTW, the middle one (which, based on your answer is not used at all?) is the Adaftruit 1979 product, which is on the BOM for v1.2

**Reply by razidan** [reply]
I think I mislabeled my image. Seems one of them is not an Adafruit product.
![headers](https://github.com/alanbjohnston/CubeSatSim/assets/78091655/637fcc33-24c9-44ff-ba7f-5bbe4774a618)
